### ISSUE TITLE
Instructions and Test do not match

### ISSUE THREAD

**Issue by catapixel** [FreeCodeCamp/FreeCodeCamp#3994]
Challenge [Waypoint: Delete Properties from a JavaScript Object](http://www.freecodecamp.com/challenges/waypoint-delete-properties-from-a-javascript-object#?solution=var%20ourDog%20%3D%20%7B%0A%20%20%22name%22%3A%20%22Camper%22%2C%0A%20%20%22legs%22%3A%204%2C%0A%20%20%22tails%22%3A%201%2C%0A%20%20%22friends%22%3A%20%5B%22everything!%22%5D%2C%0A%20%20%22bark%22%3A%20%22bow-wow%22%0A%7D%3B%0A%0Adelete%20ourDog.bark%3B%0A%0Avar%20myDog%20%3D%20%7B%0A%20%20%22name%22%3A%20%22Happy%20Coder%22%2C%0A%20%20%22legs%22%3A%204%2C%0A%20%20%22tails%22%3A%201%2C%0A%20%20%22friends%22%3A%20%5B%22Free%20Code%20Camp%20Campers%22%5D%2C%0A%20%20%22bark%22%3A%20%22woof%22%0A%7D%3B%0A%0A%2F%2F%20Only%20change%20code%20below%20this%20line.%0A%0Adelete%20myDog.tails%3B%0A%0A%2F%2F%20Only%20change%20code%20above%20this%20line.%0A%0A(function(z)%7Breturn%20z%3B%7D)(myDog)%3B%0A) has an issue.
User Agent is: <code>Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/46.0.2490.80 Safari/537.36</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

The example code `delete myDog.bark;` should be changed to `delete ourDog.bark;` to match the previous waypoint and the instructions `Let's delete the "tails" property from myDog.` does not match the test which says to `Delete the property "bark" from myDog.` My guess is that the instructions should be changed to:

`Let's delete the "bark" property from myDog.`

My code:
```javascript
var ourDog = {
  "name": "Camper",
  "legs": 4,
  "tails": 1,
  "friends": ["everything!"],
  "bark": "bow-wow"
};

delete ourDog.bark;

var myDog = {
  "name": "Happy Coder",
  "legs": 4,
  "tails": 1,
  "friends": ["Free Code Camp Campers"],
  "bark": "woof"
};

// Only change code below this line.

delete myDog.tails;

// Only change code above this line.

(function(z){return z;})(myDog);

```

**Comment by ArielLeslie** [comment]
![untitled](https://cloud.githubusercontent.com/assets/6234475/10825257/d6500cc0-7e29-11e5-89ec-7898585f11ee.png)

Adding screenshot of issue.

**Comment by ltegman** [comment]
This has been partially fixed by PR #3967 and is pending deployment to the website (the test has been changed to match instructions.

However, making the example text more consistent with the previous lessons by using ourDog instead of myDog would be good.
